data/Debian-8.yaml

```diff
@@ -0,0 +1,2 @@
+---
+nscd::service_restart: /usr/sbin/service nscd reload
```

Comment:
I know that Debian 8 could be used with systemd and sysv, but in all other modules we have we assume it's running systemd. (also I think we should not support debian 8 anymore because it has very limited support and debian 9/10 are available)